public/js/report.js

```diff
@@ -742,6 +745,24 @@ var Hit = React.createClass({
 });
 
 /**
+ * Renders Length Distribution of all hits per query
+ */
+var LengthVis = React.createClass({
+  plot_svg: function () {
```

Comment:
Rename it to `svgContainer` in line with GraphicalOverview and Kablammo components.